x/authority/types/authorization_list_test.go

```diff
@@ -422,12 +416,18 @@ func TestDefaultAuthorizationsList(t *testing.T) {
 		var AdminPolicyMessageList = []string{
 			sdk.MsgTypeURL(&crosschaintypes.MsgMigrateTssFunds{}),
 			sdk.MsgTypeURL(&crosschaintypes.MsgUpdateTssAddress{}),
+			sdk.MsgTypeURL(&crosschaintypes.MsgWhitelistERC20{}),
+			sdk.MsgTypeURL(&fungibletypes.MsgDeployFungibleCoinZRC20{}),
 			sdk.MsgTypeURL(&fungibletypes.MsgUpdateContractBytecode{}),
 			sdk.MsgTypeURL(&fungibletypes.MsgUpdateSystemContract{}),
+			sdk.MsgTypeURL(&fungibletypes.MsgRemoveForeignCoin{}),
 			sdk.MsgTypeURL(&observertypes.MsgUpdateObserver{}),
+			sdk.MsgTypeURL(&observertypes.MsgAddObserver{}),
+			sdk.MsgTypeURL(&observertypes.MsgRemoveChainParams{}),
 			sdk.MsgTypeURL(&types.MsgAddAuthorization{}),
 			sdk.MsgTypeURL(&types.MsgRemoveAuthorization{}),
 			sdk.MsgTypeURL(&types.MsgUpdateChainInfo{}),
+			sdk.MsgTypeURL(&lightclienttypes.MsgEnableHeaderVerification{}),
```

Comment:
Maybe we should link a document ( such as the notion doc ) at the top of this file

Reply:
Can you elaborate on this? Notion is private